pkg/api/mlflow/config/config.go

```diff
@@ -35,6 +36,7 @@ func NewServiceConfig() *ServiceConfig {
 		AuthPassword:          viper.GetString("auth-password"),
 		DefaultArtifactRoot:   viper.GetString("default-artifact-root"),
 		S3EndpointURI:         viper.GetString("s3-endpoint-uri"),
+		GSEndpointURI:         viper.GetString("gs-endpoint-uri"),
```

Comment:
please hide this setting, as users won't actually need it

Reply:
aha, I see your point. done.